### DISCUSSION TITLE
Model the Increase of G01

### DISCUSSION THREAD

**Discussion by Bgzh** [vkmbox/Ml-readinggroup#3] [Q&A]
If the increase of g01 is expected by the theory in the book, is it possible to predict its increasing rate?

**Comment by vkmbox** [comment]
> If the increase of g01 is expected by the theory in the book, is it possible to predict its increasing rate?

Hi @Bgzh, I think in the experiment circumstances G01, G00 and G11 should be constant.  If we check 5.45 together with 5.55 - 5.57, we realize that at criticality $K_{[1]}$ and $K_{[2]}$ stay approx. stable as well as $ΔK_{00}$; consequently, if fixing-point K* (5.8) is constant then $K_{00}$ is also constant and whole 5.45 stays stable which means $K_{xy}$ doesn't change from level to level (cov 00, cov 01 and cov 11 are stable). Also, there is analysis for relu on p. 124-125 saying that "every perturbation stays constant". Thus, I think in relu case for a preactivation at any level with any training-point pair the covariation must stay the same due to the theory. Only problem is that $K_{xx}$ is a kernel-part of $G_{xx}$, however, there is analysis on p. 140 saying that for relu with finite-width NLO is zero, consequently $K_{xx}$ is almost $G_{xx}$. Also, we need to understand Bruno's point, he did some explanations using 5.166.
